Frends.HTTP.DownloadFile/Frends.HTTP.DownloadFile/Definitions/Input.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.HTTP.DownloadFile.Definitions;
+
+/// <summary>
+/// Input class
+/// </summary>
+public class Input
+{
+    /// <summary>
+    /// The URL with protocol and path. 
+    /// You can include query parameters directly in the URL.
+    /// </summary>
+    /// <example>https://example.org/path/to</example>
+    [DefaultValue("https://example.org/path/to")]
+    [DisplayFormat(DataFormatString = "Text")]
+    public string Url { get; set; }
+
+    /// <summary>
+    /// List of HTTP headers to be added to the request.
+    /// </summary>
+    /// <example>[ foo, bar ]</example>
```

Comment:
This is not a correct example, since the Property is array of objects. Please use something like `[ { Name = "Content-Type", Value = "application/json" }, { Name = "Accept", Value = "application/json"  } ]`.